radio/util/hw_defs/switch_config.py

```diff
@@ -15,15 +15,22 @@
     },
     "gx12": {
         # left side
-        "SA": {"default": "TOGGLE", "display": [0, 0]},
-        "SB": {"default": "3POS",   "display": [0, 1]},
-        "SE": {"default": "3POS",   "display": [0, 2]},
-        "SG": {"default": "TOGGLE", "display": [0, 3]},
+        "SA": {"default": "TOGGLE"},
+        "SB": {"default": "3POS",   "display": [0, 0]},
+        "SE": {"default": "3POS",   "display": [0, 1]},
+        "SG": {"default": "TOGGLE", "display": [0, 2]},
         # right side
-        "SC": {"default": "3POS",   "display": [1, 1]},
-        "SD": {"default": "TOGGLE", "display": [1, 0]},
-        "SF": {"default": "3POS",   "display": [1, 2]},
-        "SH": {"default": "TOGGLE", "display": [1, 3]},
+        "SC": {"default": "3POS",   "display": [1, 0]},
+        "SD": {"default": "TOGGLE"},
+        "SF": {"default": "3POS",   "display": [1, 1]},
+        "SH": {"default": "TOGGLE", "display": [1, 2]},
```

Comment:
Is this change intended? If yes, what would be the reason?

Reply:
Yes. SA & SD are function switches - they have their own state display (LED) so don't need to be displayed on the LCD.